File: problems/problems_LCR_003/Solution.java
package problems.problems_LCR_003;

import com.alibaba.fastjson.JSON;
import java.util.*;
import qubhjava.BaseSolution;


public class Solution extends BaseSolution {
    public int[] countBits(int n) {

    }

    @Override
    public Object solve(String[] inputJsonValues) {
        int n = Integer.parseInt(inputJsonValues[0]);
        return JSON.toJSON(countBits(n));
    }
}

File: problems/problems_LCR_003/solution.go
package problemLCR_003

import (
	"encoding/json"
	"log"
	"strings"
)

func countBits(n int) []int {

}

func Solve(inputJsonValues string) interface{} {
	inputValues := strings.Split(inputJsonValues, "\n")
	var n int

	if err := json.Unmarshal([]byte(inputValues[0]), &n); err != nil {
		log.Fatal(err)
	}

	return countBits(n)
}
